templates/base.jinja2

```diff
@@ -11,12 +11,63 @@
     <!-- ADD THE FOLLOWING LINE ONLY -->
     <link href="/static/style.css" rel="stylesheet" type="text/css">
     <link href="/static/pygments.css" rel="stylesheet" type="text/css">
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/1.11.3/jquery.min.js"></script>
+    <script src="/static/main.js"></script>
```

Comment:
unless I'm mistaken, this `main.js` file currently contains only commented-out code.  That's not going to be particularly useful, so why bother downloading it?